library/adoptopenjdk

```diff
@@ -4,384 +4,480 @@ Maintainers: Dinakar Guniguntala <dinakar.g@in.ibm.com> (@dinogun)
 
-Tags: 8u262-b10-jre-hotspot-windowsservercore-1809, 8-jre-hotspot-windowsservercore-1809
-SharedTags: 8u262-b10-jre-hotspot-windowsservercore, 8-jre-hotspot-windowsservercore, 8u262-b10-jre-hotspot, 8-jre-hotspot
+Tags: 8u265-b01-jre-hotspot-windowsservercore-ltsc2016, 8-jre-hotspot-windowsservercore-ltsc2016
+SharedTags: 8u265-b01-jre-hotspot-windowsservercore, 8-jre-hotspot-windowsservercore, 8u265-b01-jre-hotspot, 8-jre-hotspot
 Architectures: windows-amd64
-GitCommit: 2ad867c3b57069823d3958515a55d227d3153da2
-Directory: 8/jre/windows/windowsservercore-1809
+GitCommit: deab34cef2565399c54cb4ab47fb06a54007b3f6
+Directory: 8/jre/windows/windowsservercore-ltsc2016
 File: Dockerfile.hotspot.releases.full
-Constraints: windowsservercore-1809
+Constraints: windowsservercore-ltsc2016
 
-#-----------------------------hotspot v11 images--------------------------------
 
+#-----------------------------hotspot v11 images---------------------------------
 Tags: 11.0.8_10-jdk-hotspot-bionic, 11-jdk-hotspot-bionic, 11-hotspot-bionic
 SharedTags: 11.0.8_10-jdk-hotspot, 11-jdk-hotspot, 11-hotspot
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 2ad867c3b57069823d3958515a55d227d3153da2
+GitCommit: deab34cef2565399c54cb4ab47fb06a54007b3f6
 Directory: 11/jdk/ubuntu
 File: Dockerfile.hotspot.releases.full
 
-Tags: 11.0.8_10-jdk-hotspot-windowsservercore-ltsc2016, 11-jdk-hotspot-windowsservercore-ltsc2016, 11-hotspot-windowsservercore-ltsc2016
+Tags: 11.0.8_10-jdk-hotspot-windowsservercore-1809, 11-jdk-hotspot-windowsservercore-1809, 11-hotspot-windowsservercore-1809
 SharedTags: 11.0.8_10-jdk-hotspot-windowsservercore, 11-jdk-hotspot-windowsservercore, 11-hotspot-windowsservercore, 11.0.8_10-jdk-hotspot, 11-jdk-hotspot, 11-hotspot
 Architectures: windows-amd64
-GitCommit: 2ad867c3b57069823d3958515a55d227d3153da2
-Directory: 11/jdk/windows/windowsservercore-ltsc2016
+GitCommit: deab34cef2565399c54cb4ab47fb06a54007b3f6
+Directory: 11/jdk/windows/windowsservercore-1809
 File: Dockerfile.hotspot.releases.full
-Constraints: windowsservercore-ltsc2016
+Constraints: windowsservercore-1809
 
-Tags: 11.0.8_10-jdk-hotspot-windowsservercore-1809, 11-jdk-hotspot-windowsservercore-1809, 11-hotspot-windowsservercore-1809
+Tags: 11.0.8_10-jdk-hotspot-windowsservercore-ltsc2016, 11-jdk-hotspot-windowsservercore-ltsc2016, 11-hotspot-windowsservercore-ltsc2016
 SharedTags: 11.0.8_10-jdk-hotspot-windowsservercore, 11-jdk-hotspot-windowsservercore, 11-hotspot-windowsservercore, 11.0.8_10-jdk-hotspot, 11-jdk-hotspot, 11-hotspot
 Architectures: windows-amd64
-GitCommit: 2ad867c3b57069823d3958515a55d227d3153da2
-Directory: 11/jdk/windows/windowsservercore-1809
+GitCommit: deab34cef2565399c54cb4ab47fb06a54007b3f6
+Directory: 11/jdk/windows/windowsservercore-ltsc2016
 File: Dockerfile.hotspot.releases.full
-Constraints: windowsservercore-1809
+Constraints: windowsservercore-ltsc2016
 
 Tags: 11.0.8_10-jre-hotspot-bionic, 11-jre-hotspot-bionic
```

Comment:
Arg, this isn't based on Ubuntu Bionic anymore (and we didn't catch it). :disappointed: